pkg/vars/stream_test.go

```diff
@@ -43,17 +45,21 @@ func TestReadFile(t *testing.T) {
 
 func TestHTTPContent(t *testing.T) {
 	st := Stream{}
+	wg := &sync.WaitGroup{}
```

Comment:
Hi, I'm struggeling to see why the WaitGroup is used. Could you elaborate a bit for me?
Thanks!

Reply:
It pauses the test until the http server is up and running. It communicates such when it calls the wait groups done function. 

Without it, there is a race condition between starting the server and attempting to use it.

Reply:
But I don’t see any wg.Wait() that makes the test wait as you are describing. That's why a bit lost.

Reply:
To be honest, I'm not sure exactly how it works. I'm new to go (mmock is the first go code I've ever worked with) I found that I couldn't shut down the server once started, this hanging the tests. I found this, and it works. 

https://stackoverflow.com/questions/39320025/how-to-stop-http-listenandserve